Add generate templates command (#1242)
onflow/flow-cli: README.md

Usage:
  flow [command]

👋 Welcome Flow developer!

>    If you are starting a new flow project use our super commands, start by running 'flow setup'. 

Reviewer: ```suggestion
   If you are starting a new flow project try running 'flow setup <project_name>'. 
```

> 🔥 Super Commands

Reviewer: should we include any required or option args in this list?

Author: Not necessary for this. Take a look at the home readme now. It's just out of date. Can drill into the commands for more details.

   If you are starting a new flow project use our super commands, start by running 'flow setup'. 

🔥 Super Commands
  dev          Build your Flow project
  flix         Execute FLIX template with a given id, name, or local filename
  generate     Generate new boilerplate files
  setup        Start a new Flow project

📦 Flow Entities
  accounts     Create and retrieve accounts and deploy contracts
  blocks       Retrieve blocks
  collections  Retrieve collections
  events       Retrieve events

💬 Flow Interactions
  scripts      Execute Cadence scripts
  transactions Build, sign, send and retrieve transactions

🔨 Flow Tools
  cadence      Execute Cadence code

>   collections  Utilities to read collections

Reviewer: why removing these and project commands (keys, scripts, tx, etc)?

Author: I think they just moved. This is a copy paste of what comes from the CLI typing `flow` so it's whatever commands exist.
